sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs

```diff
@@ -2221,6 +2274,83 @@ public partial class CosmosDBPrivateLinkServiceConnectionStateProperty
         public static bool operator !=(Azure.ResourceManager.CosmosDB.Models.CosmosDBServerVersion left, Azure.ResourceManager.CosmosDB.Models.CosmosDBServerVersion right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class CosmosDBServiceCreateOrUpdateContent
+    {
+        public CosmosDBServiceCreateOrUpdateContent() { }
+        public int? InstanceCount { get { throw null; } set { } }
+        public Azure.ResourceManager.CosmosDB.Models.CosmosDBServiceSize? InstanceSize { get { throw null; } set { } }
+        public Azure.ResourceManager.CosmosDB.Models.CosmosDBServiceType? ServiceType { get { throw null; } set { } }
+    }
+    public partial class CosmosDBServiceProperties
+    {
+        public CosmosDBServiceProperties() { }
+        public System.Collections.Generic.IDictionary<string, System.BinaryData> AdditionalProperties { get { throw null; } }
```

Comment:
We're discussing if just change this additionalProperties to `BinaryData` instead of `IDictionary<string, System.BinaryData>`, which is easier to assign values

Reply:
Would manual changes to this file be overwritten when the next API version is released and we generate it again?

Reply:
Keeping it as is for this PR.